Claudia/src/main/java/com/endava/claudia/task/four/CompanyOrder.java

```diff
@@ -0,0 +1,89 @@
+package com.endava.claudia.task.four;
+
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.Random;
+
+public class CompanyOrder implements Order {
+
+    public long id = 1;
+    public LocalDateTime dateTime;
+    public String company;
+    public List<Item> items;
+
+    public CompanyOrder(String company, List<Item> items) {
+        this.id = Math.abs(new Random().nextLong());
+        this.dateTime = LocalDateTime.now();
+        this.company = company;
+        this.items = items;
+    }
+
+    public boolean containsClothes() {
+        for (Item item : this.items) {
+            if (item.getType() == Type.CLOTHES) {
```

Comment:
Why only CLOTHES? Could you make this method more generic?

Reply:
Done